tap_pendo/streams.py

```diff
@@ -238,6 +243,9 @@ def send_request_get_results(self, req):
                           giveup=lambda e: e.response is not None and 400 <= e.
                           response.status_code < 500,
                           factor=2)
+    @backoff.on_exception(backoff.expo, (ConnectionError, ProtocolError), # backoff error
```

Comment:
@harshpatel4crest Where are we backing off the Timeout Error exception?

Reply:
Backoff for Timeout error is already included in `requests.exceptions.RequestException` as it is the parent class of 'Timeout' error.

Reply:
Added a comment for the same.